lib/gauth.js

```diff
@@ -38,14 +38,17 @@ async function getAuthClient(support) {
 }
 
 async function getJWTAuthClient() {
-	const content = await fs.readFile(process.env.GAPI_SERVICE_ACCOUNT_JWT);
+	const content = await fs.readFile(
+		process.env.GAPI_SERVICE_ACCOUNT_JWT,
+		'utf8',
```

Comment:
This tells nodejs to treat the file as utf8 so it'll return a string instead of a buffer, previously this conversion was happening implicitly at the `JSON.parse` step and this just makes it explicit so it can satisfy type-checks